server/global-types.ts

```diff
@@ -2,6 +2,21 @@ type ChildProcess = import('child_process').ChildProcess;
 
 type Config = typeof import('../config/config-example') & AnyObject;
 
+type ServerGroup =
+	| '~'
```

Comment:
Why are these all on their own lines, anyway? Seems unnecessarily verbose.

Reply:
Also, even if TypeScript allows a leading `|`, I don't think it's very readable.

Reply:
I thought collapsing them into one line would make it long, but it turned out to be 90 col so it's OK.